simulation/Jamfile

```diff
@@ -18,8 +18,6 @@ project
 	<source>utils.cpp
 	: default-build
 	<threading>multi
-# disable warning C4275: non DLL-interface classkey 'identifier' used as base for DLL-interface classkey 'identifier'
-	<toolset>msvc:<cflags>/wd4275
```

Comment:
ah, this is what was causing the failure.. because it was under default-build, rather than requirements (if there are warnings when building the simulation, you could probably add it back there)